docs/bugs_and_glitches.md

```diff
@@ -2065,6 +2066,24 @@ Most of the NPCs in [maps/NationalParkBugContest.asm](https://github.com/pret/po
 (The use of `SPRITE_ROCKER` instead of `SPRITE_COOLTRAINER_M` for `COOLTRAINERM NICK` may also be an intentional reference to the player's brother from the [Space World '97 beta](https://github.com/pret/pokegold-spaceworld).)
 
 
+### `BattleAnim_Tackle` reaches horizontal sprite limit
+
+`BattleAnim_Tackle` in Pokémon Crystal reaches the horiztonal sprite limit due to two rows of the enemy's feet being copied from background tiles to sprites. This fix restores the animation to only copy one row like in Pokémon Gold and Silver.
+
+![image](https://raw.githubusercontent.com/pret/pokecrystal/master/docs/images/tackle_animation_bug.png)
```

Comment:
Why is there a picture?

Reply:
Why are there videos in others? Its just to _show_ what the bug is. I know we don't do it now elsewhere; but I don't think its a terrible idea. This is just my way to suggest it; it can be removed if you'd rather not.

Reply:
There aren't videos in others; we just link to them because they existed already. This minor bug doesn't need an illustration unless you feel like adding them for every bug (and I recommend just not adding this).

Reply:
I agree with Rangi.